src/Microsoft.Extensions.Localization/Internal/IResourceStringManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace Microsoft.Extensions.Localization.Internal
+{
+    public interface IResourceStringManager
+    {
+        string GetResourceName(CultureInfo culture);
+
+        IList<string> GetAllResourceStrings(CultureInfo culture);
```

Comment:
I can accomplish this by making it the `IResourceStringManager`'s job to throw when it's missing the resources, it'll be in my next pass.